portal/auth.py

```diff
@@ -0,0 +1,83 @@
+import functools
+
+from flask import (
+    Flask, Blueprint, flash, g, redirect, render_template, request, session, url_for
+)
+
+from werkzeug.security import check_password_hash, generate_password_hash
+
+from portal.db import get_db
+
+bp = Blueprint('auth', __name__)
+
+@bp.route('/', methods=('GET', 'POST'))
+def index():
+    if request.method == 'POST':
+        email = request.form['email']
+        password = request.form['password']
+        db = get_db()
+        cur = db.cursor()
+        error = None
+        cur.execute(
+            'SELECT * FROM users WHERE email = %s', (email,)
+        )
+        user = cur.fetchone()
+        cur.close()
+        print(user)
```

Comment:
These print statements used for debugging should probably not be included in the final code